lib/vehicle.rb

```diff
@@ -22,4 +28,18 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def set_plate_type
```

Comment:
Nice work using a setter method here rather than giving other classes permission to mutate your :plate_type attribute. This is a good example of encapsulation in object oriented programming